src/index.ts

```diff
@@ -18,33 +18,33 @@ const _parseCookie = (cookie: string) => {
 };
 
 const _stringToBool = (boolString: string) => {
-  if (boolString.toLowerCase() === "true" || boolString === "1") {
-    return true;
-  }
-
-  return false;
+  return boolString.toLowerCase() === "true" || boolString === "1"
 };
 
 const getFeatureFromCookie = (
-  cookieName: string,
+  key: string,
   {
     context,
     htmlDocument,
   }: {
     context?: Next.GetServerSidePropsContext;
     htmlDocument: { cookie: any };
   }
-) => {
-  const fromCookie = isServer()
-    ? context?.req.cookies[cookieName]
-    : _parseCookie(htmlDocument.cookie)[cookieName];
-
-  if (fromCookie) {
-    return _stringToBool(fromCookie);
-  }
-
-  return false;
-};
+) => isServer() ? context?.req.cookies[key] : _parseCookie(htmlDocument.cookie)[key];
+
+const getFeatureFromEnv = (key: string) => {
+  const { serverRuntimeConfig, publicRuntimeConfig } = getConfig() || {};
+
+  /*
+   * Question: should this instead be:
+   *
+   * serverRuntimeConfig[key] ?? publicRuntimeConfig[key]
+   *
+   * Returns publicRuntimeConfig[key] if serverRuntimeConfig[key] is null or undefined
+   * Returns serverRuntimeConfig[key] if is 0 or ""
+   */
+  return serverRuntimeConfig[key] || publicRuntimeConfig[key];
```

Comment:
**question:** should we use `??` instead of `||` here?

`??` would returns `publicRuntimeConfig[key]` if `serverRuntimeConfig[key]` is `null` or `undefined`

If `serverRuntimeConfig[key]` if is `0` or `""`, it returns those instead.

Reply:
Not sure, but I think it makes sense